jupyter/minimal/ubi9-python-3.11/start-notebook.sh

```diff
@@ -8,6 +8,9 @@ if [ -f "${SCRIPT_DIR}/utils/setup-elyra.sh" ]; then
   source ${SCRIPT_DIR}/utils/setup-elyra.sh
 fi
 
+export PATH=/usr/local/texlive/2025/bin/x86_64-linux:$PATH
```

Comment:
> Any suggestion on where to set the path? I am currently doing it in start-notebook.sh

Could be set in Dockerfile itself, but it appears we don't do it anywhere else this way.

Alternatively, leave this and let the test set its PATH anew, like in

https://github.com/opendatahub-io/notebooks/blob/6cd9ddb8dfc226e673fb229d12238159900806ef/tests/containers/workbenches/rstudio/rstudio_test.py#L96